pom.xml

```diff
@@ -136,10 +136,12 @@
     <maven.compiler.release>8</maven.compiler.release>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
-    <maven.plugin-version>3.0.5</maven.plugin-version>
+    <maven.plugin-version>3.5.0</maven.plugin-version>
     <!-- surefire/failsafe plugin option -->
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
     <powermock.version>2.0.2</powermock.version>
+    <!-- timestamp for reproducible outputs, updated on release by the release plugin -->
+    <project.build.outputTimestamp>2020-08-18T00:00:00Z</project.build.outputTimestamp>
```

Comment:
Is this an artifact of release testing?

Reply:
This property is for reproducible builds. The timestamp is used by several plugins where timestamps would have caused the plugin to produce different binaries. See https://maven.apache.org/guides/mini/guide-reproducible-builds.html

The property itself is supposed to be updated in the POM by the release plugin when it updates the version number during the release process. I have not tested, but will look for that during RC testing when it comes to that.

Reply:
Ok.  I thought maybe you were testing the maven release process and this was accidentally introduced.